src/Microsoft.DotNet.Darc/src/DarcLib/Actions/Remote.cs

```diff
@@ -352,12 +363,64 @@ public async Task DeletePullRequestBranchAsync(string pullRequestUri)
             }
         }
 
-        public async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestParameters parameters)
+        /// <summary>
+        ///     Parse out the owner and repo from a repository url
+        /// </summary>
+        /// <param name="pattern">Regex Pattern</param>
+        /// <param name="prBody">Pull Request commit msg</param>
+        /// <returns>Tuple of owner and repo</returns>
+        public string ParsePullRequestBody(Regex pattern, string prBody)
```

Comment:
Why not just... leave the `**`, then this whole method doesn't need to exist. It's just pattern.Matches(prBody).Select(m => m.Value).  There's a lot of... strange stuff in here, and the simpler regex code is, the better.

Reply:
The comments are also pretty misleading, it doesn't do anything like that.

Reply:
done